.ci/azure/linux_arm64.yml

```diff
@@ -63,66 +104,235 @@ jobs:
     submodules: recursive
     path: openvino
 
+  - checkout: open_model_zoo
+    clean: true
+    lfs: false
+    submodules: recursive
+    path: open_model_zoo
+
   - script: |
       set -e
-      $(OPENVINO_REPO_DIR)/install_build_dependencies.sh
+      $(REPO_OPENVINO)/install_build_dependencies.sh
       # Move into contrib install_build_dependencies.sh
-      sudo apt --assume-yes install scons crossbuild-essential-arm64 libprotoc-dev protobuf-compiler
-      # OpenCV should provide install_build_dependencies.sh as well
-      # Move into resources
-      git clone https://github.com/opencv/opencv.git --depth 1 $(OPENCV_REPO_DIR)
+      sudo apt --assume-yes install scons crossbuild-essential-arm64 libprotoc-dev libhiredis-dev
+      sudo apt --assume-yes install protobuf-compiler default-jdk libssl-dev zip libzstd-dev python-dev
       # Speed up build
       wget https://github.com/ninja-build/ninja/releases/download/v1.10.2/ninja-linux.zip
       unzip ninja-linux.zip
       sudo cp -v ninja /usr/local/bin/
+      # OpenCV should provide install_build_dependencies.sh as well
+      # Move into resources
+      git clone https://github.com/opencv/opencv.git --depth 1 $(REPO_OPENCV)
     workingDirectory: $(WORK_DIR)
     displayName: 'Install dependencies'
 
+  - script: |
+      #### Build python for host 
+      wget https://www.python.org/ftp/python/$(PYTHON_ARM_VERSION)/Python-$(PYTHON_ARM_VERSION).tgz
```

Comment:
should be a part of contrib install_build_dependencies.sh

Reply:
done